.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -380,6 +383,22 @@ jobs:
             ./core/deps/llvm/build/share
           key: ${{ matrix.llvm_cache_key }}
 
+      - name: configure ubsan env
+        if: matrix.sanitizer == 'ubsan'
```

Comment:
depending on how much work is to fix all the sanitizers, you might want to just enable one of them in the first PR, and then have separate ones for adding other sanitizers and fixing issues reported by them

Reply:
Yeah, that might be the case. Thank you